integration/src/wallets/metamask.ts

```diff
@@ -83,7 +87,9 @@ export function selfTest(get: () => core.HDWallet): void {
 
   it("uses correct eth bip44 paths", () => {
     if (!wallet) return;
-    [0, 1, 3, 27].forEach((account) => {
+    // MM doesn't support multi-account
+    const accountNumbers = metamask.isMetaMask(wallet) ? [0] : [0, 1, 3, 27];
```

Comment:
this intergration test is only for metamask anyways, so ternary is kind of pointless. can just have `[0]` if we don't want to show that it still _is_ able to derive account paths even if not natively supported by the wallet itself

Reply:
e0e8f4e5